plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/adapter/controllers/CDKLaunchController.java

```diff
@@ -154,7 +154,14 @@ public void launch(ILaunchConfiguration configuration, String mode, ILaunch laun
 		}
 		
 		String args = configuration.getAttribute(ATTR_ARGS, (String)null);
-		ILaunchConfigurationWorkingCopy wc = new VagrantLaunchUtility().createExternalToolsLaunchConfig(s, args, getStartupLaunchName(s));
+		ILaunchConfigurationWorkingCopy wc = null;
+		try {
+			wc = new VagrantLaunchUtility().createExternalToolsLaunchConfig(s, args, getStartupLaunchName(s));
+		} catch(CoreException ce) {
+			CDKCoreActivator.pluginLog().logError(ce);
```

Comment:
For CDK server? Currently having credentials is a hard requirement. Especially in the openshift-vagrant repository. But yes, it's likely this will be re-visited when the raw cdk is supported.